app/src/main/java/org/p2p/wallet/common/ui/widget/CustomSnackBar.kt

```diff
@@ -40,17 +39,15 @@ class CustomSnackBar @JvmOverloads constructor(
         animatorSet.start()
     }
 
-    override fun animateContentOut(delay: Int, duration: Int) = NoOp
+    override fun animateContentOut(delay: Int, duration: Int) = Unit
 
-    fun setMessage(text: String): CustomSnackBar {
+    fun setMessage(text: String): CustomSnackBar = apply {
         binding.textView.text = text
-        return this
     }
 
-    fun setIcon(@DrawableRes iconResId: Int): CustomSnackBar {
+    fun setIcon(@DrawableRes iconResId: Int): CustomSnackBar = apply {
```

Comment:
not obvious at all )

Reply:
very popular pattern to update instance and return it, i saw it in our codebase